index.html

```diff
@@ -0,0 +1,261 @@
+<html>
+<head>
+  <style>
+  table {
+      border-collapse: collapse;
+      table-layout: fixed;
+  }
+  
+  table, th, tr, td{
+      border: 1px solid black;
+      height: 30px;
+  }
+  
+  table, th{
+      width: 100%;
+      font-size: 25px;
+  }
+  
+  th, a{
+     cursor: default; 
+  }
+  
+  .inputField {
+      width: 100%;
+      padding: 0px;
+      border-width: 0;
+      height: 100%;
+      font-size: 17px;
+  }
+  
+  .btn{
+      text-align: center;
+      vertical-align: middle;
+      cursor: default;
+  }
+  
+  .center {
+      margin-bottom: 8px;
+  }
+  
+  .checkbox{
+      cursor: default;
+  }
+  
+  [name=DateCell], [name=DeleteCell]  {
+      font-size: initial;
+  }
+  
+  #addRowBtn {
+      height: 25px;
+      font: icon;
+  }
+  </style>
+</head>
+
+<body onload='myTable()'>
+<script>
+  function myTable() {
+      'use strict';
+      
+      var tbl = [],
+          sortASC = [true, true, true, true, true],
+          addRowBtn = document.getElementById('addRowBtn'),
+          headers = document.getElementsByClassName('label'),
```

Comment:
This means that you have to click on the text to trigger the sorting, clicking on the cell will not. Maximizing clickable area is good practice. (when you click on a button, you can click on anywhere on the button, not just the text of the button)